Community board blog post Q1 2024 (#2092)
writethedocs/www: docs/blog/2024-Q1-community-board.rst

> .. post:: Feb 1, 2024

Reviewer: ```suggestion
.. post:: Feb 12, 2024
```

   :tags: community board, news, report
   :author: Mikey Ariel

Write the Docs Community Board Quarterly Update Q1-2024
=======================================================

> Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share with you the first quarterly update for 2024, the beginning of our second year of operation. 

Reviewer 2: Nit: Disambiguate "second year of operation" to mean second year since forming the board?

> Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share with you the first quarterly update for 2024, the beginning of our second year of operation. 

Author: ```suggestion
Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share with you the first quarterly update for 2024, which marks the beginning of our second year of running the board. 
```

To catch up on previous quarterly updates, check out the `community board blog posts <https://www.writethedocs.org/blog/archive/tag/community-board/>`_ archive. 

Q1 quarterly meeting highlights
-------------------------------

> Our Q1 meeting included an informal retrospective on 2023, which was our first year of operation. As a newly grouped collection of team leads, we dedicated some of our efforts to exploring ways to collaborate with each other more proactively and evolving our meta-work tracking and priorities.

Reviewer 2: Opinion: I don't love "grouped" but don't have an alternative.

> Our Q1 meeting included an informal retrospective on 2023, which was our first year of operation. As a newly grouped collection of team leads, we dedicated some of our efforts to exploring ways to collaborate with each other more proactively and evolving our meta-work tracking and priorities.

Author: ```suggestion
Our Q1 meeting included an informal retrospective on 2023, which was our first year of running the board. Since this was the first year of meeting regularly with all the team leads, we started by taking some time to exploring ways to collaborate with each other more proactively and evolving our meta-work tracking and priorities.
```

> Some of the activities that our board members focused on in 2023:

Author: ```suggestion
Some of the activities that the community board focused on as a group in 2023:
```

> * Reviewed and organized our Kanban board that is used for tracking community-wide task

Reviewer 2: I think activities that our members focussed on should probably all be "reviewing and organzing", "launching", etc.

Author: I was referring to the community board activities rather than individual member activities, clarified in the intro sentence

Reviewer 2: I get that. But the tense still reads wrong to me. Never mind.

> * Reviewed and organized our Kanban board that is used for tracking community-wide task

Author: ```suggestion
* Reviewed and organized our Kanban board that is used for tracking community-wide tasks and long-term projects
```

---------------------------

The `Community Board <https://www.writethedocs.org/team/#community-board>`_ exists to curate the overall strategic direction of the community teams.

The board members are team leads for each WTD team, as well as the conference chairs of our various conferences:

* Mikey Ariel (CoreOps, brand and design, board chair)
* Eric Holscher (CoreOps, Portland chair)
* Samuel Wright (CoreOps, infrastructure)
* Sasha Romijn (CoreOps, support, Atlantic chair)
* Rose Williams (meetups)
* Janine Chan (Slack moderation)
* Aaron Collier (newsletter)
* Swapnil Ogale (Australia chair)

The community board meets quarterly in real-time, and provides monthly asynchronous status updates. Board members can share the status of what they're working on and request any help to keep the various teams functioning effectively.

Thank you 
---------

Thanks again to the team leads that participate in the community board, and to all the team members (mostly volunteers!) who work diligently to support the community and each other. If you want to join any of the WTD community teams, check out the `Team page <https://www.writethedocs.org/team/>`_ for team descriptions and contacts. 

> See you next year for the first quarterly update of 2024!

Author: ```suggestion
See you in the next update, and wishing everyone a joyous and productive 2024!
```
